src/main/scala/org/neo4j/spark/Neo4jOptions.scala

```diff
@@ -188,6 +231,8 @@ object Neo4jOptions {
   val DEFAULT_ENCRYPTION_ENABLED = false
   val DEFAULT_ENCRYPTION_TRUST_STRATEGY = TrustStrategy.Strategy.TRUST_SYSTEM_CA_SIGNED_CERTIFICATES
   val DEFAULT_SCHEMA_FLATTEN_LIMIT = 10
+  val DEFAULT_BATCH_SIZE = 500
```

Comment:
depends on node size of course but as a default I'd go much higher, like 5k.   Typical values we use as a starting point in some tests are 20k, but these can be dangerously high and blow up heap if nodes are fat.